pkg/provisioner/config/config.go

```diff
@@ -0,0 +1,131 @@
+package config
+
+import (
+	"context"
+	"fmt"
+
+	hypersdsv1alpha1 "github.com/tmax-cloud/hypersds-operator/api/v1alpha1"
+	"github.com/tmax-cloud/hypersds-operator/pkg/common/wrapper"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// CephConfig is struct for ceph config
+type CephConfig struct {
+	crConf    map[string]string
+	admConf   map[string]string
+	admSecret map[string][]byte
+}
+
+// SetCrConf sets crConf to value
+func (conf *CephConfig) SetCrConf(m map[string]string) error {
```

Comment:
By our naming rule, it should be `SetCRConf`

Reply:
We decided to do so only when necessary